server/config/default.json

```diff
@@ -34,7 +34,8 @@
     "adminShow": false,
     "HideCapcode": false,
     "HideSource": false,
-    "apiSecurity": false
+    "apiSecurity": false,
+    "accordionEnable": true
```

Comment:
disable by default is best i think... shouldn't affect upgrades but still i prefer to let people opt-in rather than out 😁  @pagermon/collaborators ?

Reply:
I agree